devices/device.go

```diff
@@ -0,0 +1,12 @@
+package devices
+
+import "errors"
+
+var errDataLenInvalid = errors.New("invalid data size on port")
+
+type Device interface {
```

Comment:
I understand that this is a device connected in a way other than PCI. Can you leave a comment about that?
I would like to clean up the Device interface in pci.go and device.go in the future.